### DISCUSSION TITLE
Short text on timer

### DISCUSSION THREAD

**Discussion by Nickyes** [AntonyCorbett/OnlyT#446] [Ideas]
Very useful tool is to send a short message on timer monitor, so than the speaker or chairman can get a brief technical message or other informations. Better if this message don't cover timer

**Comment by AntonyCorbett** [comment]
@Nickyes Thanks. Please See #412 and #39

**Reply by Nickyes** [reply]
saw that there you marked as enhancement. thanks

**Reply by Wistula** [reply]
If you are interested in solving the topic indirectly, take a look at #412 . I added my idea there. I don't know what brother Antony would say to use Only V in this way, I hope he won't mind. We are very grateful to you for the amount of your work.

**Reply by Nickyes** [reply]
I answered there, but i think didn't fully understand my request. Anyway thanks :)
